File: CodeForces/1-100/59/A/59A.go
// Problem 59A: Word
// https://codeforces.com/contest/59/problem/A
// Executed with: go version go1.24.4 linux/arm64
// Linux version: Linux kernel 6.17.0-5-generic
// Submitted on: October 26th, 2025
// Codeforces language used: Go 1.22.2

package main

import (
	"fmt"
	"strings"
)

func main() {
	var input_str string
	fmt.Scan(&input_str)
	var running int
	for i := 0; i < len(input_str); i++ {
		var to_check string = string(input_str[i])
		if to_check == strings.ToUpper(to_check) {
			running += 1
		} else {
			running -= 1
		}
	}

	if running > 0 {
		fmt.Println(strings.ToUpper(input_str))
	} else {
		fmt.Println(strings.ToLower(input_str))
	}
}

File: CodeForces/1-100/59/A/59A.py
# Problem 59A: Word
# https://codeforces.com/contest/59/problem/A
# Executed with: Python 3.11.2
# Linux version: Linux kernel 6.1.0-10-amd64
# Submitted on: September 27th, 2024
# Codeforces language used: PyPy 3.10 (7.3.15, 64bit)

in_str = input()

upper = 0
lower = 0

for char in in_str:
    if char.isupper():
        upper += 1
    else:
        lower += 1

if upper > lower:
    print(in_str.upper())
else:
    print(in_str.lower())
